Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [InnerTune/App - Main](https://hosted.weblate.org/projects/innertune/app/).


It also includes following components:

* [InnerTune/App - Fastlane](https://hosted.weblate.org/projects/innertune/fastlane/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/innertune/app/horizontal-auto.svg)

lucasmz-dev commented:
Use #1827 instead for the next merge
Includes a few extra fixes

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

Commit:
Added translation using Weblate (Hebrew)

Commit:
Translated using Weblate (Hebrew)

Currently translated at 2.7% (7 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

MetalDroid committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 3.9% (10 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 98.0% (247 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/uk/

Commit:
Translated using Weblate (Czech)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/cs/

lucasmz-dev committed:
Translated using Weblate (Czech)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/cs/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/uk/

niko22493 committed:
Translated using Weblate (Spanish)

Currently translated at 97.6% (246 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

trunars committed:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/bg/

Commit:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/zh_Hans/

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/et/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fr/

lucasmz-dev committed:
Translated using Weblate (French)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fr/

lucasmz-dev committed:
Added translation using Weblate (Spanish (American))

lucasmz-dev committed:
Translated using Weblate (Spanish (American))

Currently translated at 40.0% (101 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es_US/

lucasmz-dev committed:
Translated using Weblate (Spanish (American))

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/es_US/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/pt_BR/

lucasmz-dev committed:
Translated using Weblate (Spanish (American))

Currently translated at 39.6% (100 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es_US/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fr/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

lucasmz-dev committed:
Added translation using Weblate (English (Canada))

kangurek-kao committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pl/

kangurek-kao committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/pl/

Commit:
Translated using Weblate (Japanese)

Currently translated at 98.0% (247 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ja/

Commit:
Translated using Weblate (Vietnamese)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/vi/

mikooomich committed:
Translated using Weblate (English (Canada))

Currently translated at 0.7% (2 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/en_CA/

Marxally committed:
Added translation using Weblate (Catalan)

Marxally committed:
Translated using Weblate (Catalan)

Currently translated at 19.4% (49 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ca/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 4.7% (12 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

KAMELCOLTD committed:
Translated using Weblate (Arabic)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ar/

devdevdevdevdevdev-user committed:
Translated using Weblate (Portuguese)

Currently translated at 98.0% (247 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt/

fitojb committed:
Translated using Weblate (Catalan)

Currently translated at 81.7% (206 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ca/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

lucasmz-dev commented:
Thanks for the comments. We actually need that because we don't know every language and their features. I did do that previously (it was broken because Weblate did not handle plurals properly, and because of the same thought) 

I then thought: "Well, I have when developers put pt and pt-BR together because they're annoyingly not similar enough, so I shouldn't do this". Thing is that Android fallbacks to the nearest language. Either by the languages you have set in the system, or different variants of the same language. So, when there's a missing string in es-US, it should pick up the regular es one. But yeah it does still have maintainability issues.

Spanish does have significantly different wording and stuff between regions. So I'm still not sure.

es-US did not get completed either which isn't great, and it doesn't seem like they just left stuff for fallback rather just did not do it as it was in order.

I believe Spanish may have some issues with inconsistency which I do notice in pt-BR as well in multiple places as different translators modify different things at different times and have different wording for things

ViYomX committed:
Translated using Weblate (Hindi)

Currently translated at 60.7% (153 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hi/

fitojb committed:
Translated using Weblate (Catalan)

Currently translated at 84.9% (214 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ca/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

lucasmz-dev committed:
Translated using Weblate (Hindi)

Currently translated at 60.7% (153 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hi/

lucasmz-dev commented:
@fitojb would you still like for that removal?

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/it/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fr/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/de/

CiccioDerole committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/it/

MetalDroid committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

MetalDroid committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

Commit:
Added translation using Weblate (Norwegian Nynorsk)

vishussingh24 committed:
Translated using Weblate (Hindi)

Currently translated at 65.0% (164 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hi/

ruffi-lo committed:
Translated using Weblate (German)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/de/

MetalDroid committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

ruffi-lo committed:
Translated using Weblate (German)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/de/

ruffi-lo committed:
Translated using Weblate (German)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/de/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

Commit:
Added translation using Weblate (Telugu)

Commit:
Translated using Weblate (Telugu)

Currently translated at 31.7% (80 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Turkish)

Currently translated at 97.2% (245 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/tr/

vishussingh24 committed:
Translated using Weblate (Hindi)

Currently translated at 67.0% (169 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hi/

NanakuWizard committed:
Added translation using Weblate (Undetermined)

NanakuWizard committed:
Translated using Weblate (Spanish (American))

Currently translated at 48.0% (121 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es_US/

lucasmz-dev committed:
Deleted translation using Weblate (Undetermined)

yeongaori committed:
Translated using Weblate (Korean)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ko/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ta/

rekt0ro committed:
Translated using Weblate (Persian)

Currently translated at 81.3% (205 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fa/

rekt0ro committed:
Translated using Weblate (Persian)

Currently translated at 81.3% (205 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fa/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ru/

ONGKYZ committed:
Translated using Weblate (Indonesian)

Currently translated at 96.0% (242 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/id/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/he/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Telugu)

Currently translated at 99.2% (250 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

Commit:
Translated using Weblate (Catalan)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ca/

Commit:
Translated using Weblate (Telugu)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/te/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ja/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ja/

Valer100 committed:
Added translation using Weblate (Romanian)

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ja/

MetalDroid committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ro/

Commit:
Translated using Weblate (Telugu)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/te/

dominikfrolik6-lab committed:
Translated using Weblate (English (Canada))

Currently translated at 0.7% (2 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/en_CA/

lucasmz-dev committed:
Deleted translation using Weblate (Norwegian Nynorsk)

lucasmz-dev committed:
Deleted translation using Weblate (Swedish)

lucasmz-dev committed:
Deleted translation using Weblate (Nepali)

lucasmz-dev committed:
Deleted translation using Weblate (Odia)

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Fjuro committed:
Translated using Weblate (Czech)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/cs/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ro/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fr/

Commit:
Added translation using Weblate (Slovenian)

mdvhimself committed:
Translated using Weblate (Hungarian)

Currently translated at 78.5% (198 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hu/

Ricky-Tigg committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fi/

Commit:
Translated using Weblate (Slovenian)

Currently translated at 11.9% (30 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/sl/

lucasmz-dev committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt/

mdvhimself committed:
Translated using Weblate (Hungarian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hu/

Commit:
Translated using Weblate (Slovenian)

Currently translated at 14.2% (36 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/sl/

KuzmaZmaku committed:
Added translation using Weblate (Serbian)

Commit:
Added translation using Weblate (Khmer (Central))

Commit:
Translated using Weblate (Serbian (Latin script))

Currently translated at 96.8% (244 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/sr_Latn/

Commit:
Added translation using Weblate (Basque)

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

Commit:
Translated using Weblate (Catalan)

Currently translated at 96.0% (242 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ca/

SageTendo committed:
Added translation using Weblate (Shona)

lucasmz-dev committed:
Deleted translation using Weblate (Basque)

lucasmz-dev committed:
Deleted translation using Weblate (Khmer (Central))

lucasmz-dev committed:
Deleted translation using Weblate (Serbian)

lucasmz-dev committed:
Deleted translation using Weblate (Shona)
